Cartfile

```diff
@@ -1 +1 @@
-github "ReactiveCocoa/ReactiveCocoa" "swift-development"
+github "ReactiveCocoa/ReactiveCocoa" "6c37b5ef416799a3a2a61ff78f051d5deb0d76b2"
```

Comment:
Additionally, `Cartfile.resolved` and the submodule reference should be updated too. You could use `$ carthage update --use-submodules --no-build --no-use-binaries`.